### PR TITLE
Build input component

### PR DESCRIPTION
**Pull request by ferassmohana**
The name of this component is CustomInput, and it consists of two files:

- CustomInput.jsx
- CustomInput.module.css

This component gives us the advantage of having input with the common styles, and also it gives us the chance to add icons or label.

### CODE REVIEW

File: src/components/CustomInput/CustomInput.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import styles from './CustomInput.module.css';
+
+export const CustomInput = ({ ...props }) => {
+    return (
+        <div className={styles.input_wrapp}>
```
Comment (mahmoud-alragabi) [inline src/components/CustomInput/CustomInput.jsx]: don't use snake case in naming classes use camel case

File: src/components/CustomInput/CustomInput.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import styles from './CustomInput.module.css';
+
+export const CustomInput = ({ ...props }) => {
```
Comment (mahmoud-alragabi) [inline src/components/CustomInput/CustomInput.jsx]: why are you spreading props into a new object then using it, you should extract the individual props like this ```{ label }```

File: src/components/CustomInput/CustomInput.module.css
Comment (mahmoud-alragabi) [inline src/components/CustomInput/CustomInput.module.css:1]: again use camel case

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

File: src/components/CustomInput/CustomInput.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    return (
+        <div className={styles.input_wrapp}>
+            {props.label &&
+                <label for={props.name} className={styles.input_label}>
```
Comment (mahmoud-alragabi) [inline src/components/CustomInput/CustomInput.jsx]: can't use for in jsx, replace it with htmlFor

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

File: src/components/CustomInput/CustomInput.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+                    <>
+                        {props.icon}
+                    </>
+                }
+                <input
+                    type={props.type}
+                    placeholder={props.placeholder}
+                    value={props.value}
+                    onChange={props.onChange}
+                    name={props.name}
```
Comment (mahmoud-alragabi) [inline src/components/CustomInput/CustomInput.jsx]: label for is bound to element id attribute not the name attribute, you should replace name with id here

refer to this: https://www.w3schools.com/tags/att_label_for.asp

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

**Review by ileenshaar — APPROVED** [review]

**Pull request merged**